firmware_io/simulink_input.csv

```diff
@@ -1,2 +1,51 @@
-Variable Name, Datatype, Minimum Value, Maximum Value, Name Description, Value Description,
-DI_V_SteeringAngle, float,-1,1, Steering wheel angle," ""-1 is full left turn"," +1 is full right turn. THIS IS JUST AN EXAMPLE"""
+Variable Name,Datatype,Unit,Minimum Value,Maximum Value,Name Description,Value Description,
+DI_SteeringAngle,float,%,0,1,Steering wheel angle,0 is full left turn,1 is full right turn
+DI_AccelPedalPos,float,%,0,1,Accelerator pedal position,0 is released,1 is fully pressed
+DI_BrakePedalPos,float,%,0,1,Brake pedal position,0 is released,1 is fully pressed
+DI_StartButton,bool,,0,1,Start button on dash (active low),0 is released,1 is pressed
+VD_FrtWheelSpd,int,rpm,-2000,2000,Average wheel speed of the front wheels,,
+BM_HvilFeedback,bool,,0,1,HVIL status,0 is open,1 is closed
+BM_prechrgContactorSts,bool,,0,1,Precharge contactor status,0 is open,1 is closed
+BM_HVposContactorSts,bool,,0,1,HV positive contactor status,0 is open,1 is closed
+BM_HVnegContactorSts,bool,,0,1,HV negative contactor status,0 is open,1 is closed
+BM_PackSOC,uint,%,0,100,HV battery state of charge,0 is empty,100 is full
+BM_LowThermValue,int,C,-128,127,Lowest thermistor value reported by TMS,,
+BM_HighThermValue,int,C,-128,127,Highest thermistor value reported by TMS,,
```

Comment:
if a temperature is above 127, should we clip it? 
if you mean int8, please specify

Reply:
It is the temperature reported by TMS in signed 8 bit which in theory should never go beyond 127